modules/BTBidAdapter.md

```diff
@@ -0,0 +1,40 @@
+# Overview
+
+**Module Name**: BT Bidder Adapter
+**Module Type**: Bidder Adapter  
+**Maintainer**: engsupport@blockthrough.com
+
+# Description
+
+The BT Bidder Adapter provides an interface to the BT Service. The BT Bidder Adapter sends one request to the BT Service per ad unit, but the BT Service will then send one request to each configured exchange. This is functionally similar to Prebid Server. One client side request, bids from many exchanges.
+
+The BT service accepts industry standard OpenRTB 2.5 as its payload. The BT Bidder Adapter converts Prebid.js parameters into an OpenRTB 2.5 request.
+
+# Params
+
+- `ab` required, whether AdBlock is enabled.
+- `siteId` required, site id.
```

Comment:
would be nice to specify the types, table view might work better:
<img width="670" alt="image" src="https://github.com/blockthrough/Prebid.js/assets/30797461/c259687a-7e1c-45f9-9564-723d241f464c">